code/game/area/areas/shuttles.dm

```diff
@@ -48,7 +48,7 @@
 /area/shuttle/transit
 	name = "Hyperspace"
 	desc = "Weeeeee"
-	dynamic_lighting = DYNAMIC_LIGHTING_DISABLED
+	//dynamic_lighting = DYNAMIC_LIGHTING_DISABLED //Different from /tg/
```

Comment:
Instead of commented commented-out code, can you just set dynamic_lighting to the default with the comment to explain that its being explicit?

Reply:
Sure.